client/src/styles/ProfileCard/ProfileCard.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Black+Ops+One&family=Harmattan&display=swap');
```

Comment:
You can lt ta import coz that has been already imported in the `_globals.scss` file